cvmfs/manifest.cc

```diff
@@ -114,7 +118,8 @@ string Manifest::ExportString() const {
     "R" + root_path_.ToString() + "\n" +
     "D" + StringifyInt(ttl_) + "\n" +
     "S" + StringifyInt(revision_) + "\n" +
-    "G" + StringifyBool(garbage_collectable_) + "\n";
+    "G" + StringifyBool(garbage_collectable_) + "\n"
```

Comment:
Knit picking: That is missing a `+` in the end of the line. Doesn't matter for compilation, but maybe for the sake of consistency?